public/app/features/panel/panel_ctrl.ts

```diff
@@ -99,7 +101,7 @@ export class PanelCtrl {
     }
   }
 
-  addEditorTab(title, directiveFn, index?, icon?) {
+  addEditorTab(title: string, directiveFn: any, index?: number, icon?: undefined) {
```

Comment:
It looks like we're not using icon anymore. Should we remove it?

Reply:
@hugohaggmark Any input on this?

Reply:
Maybe change to any?

```suggestion
  addEditorTab(title: string, directiveFn: any, index?: number, icon?: any) {
```

Reply:
yea, not used anymore I think. external plugins could use it I think, but not shown in UI anymore. but keep for now